docs/usage.rst

```diff
@@ -56,6 +56,8 @@ The main idea of this script is to collect the information from console.redhat.c
       - This option will show some information regarding to the user who requested the token
     * - crhc ts dump 
       - Export the whole Inventory and Subscription information in json format. Some files will be created.
+    * - crhc ts dump_current
```

Comment:
Hey @starnold-redhat 

Here is one example of piece of code that was already merged.